go.mod

```diff
@@ -10,7 +10,8 @@ require (
 	github.com/open-cluster-management/applier v0.0.0-20210323134919-2b85eca20f76
 	github.com/open-cluster-management/library-go v0.0.0-20210325215722-d989f79194f6
 	github.com/openshift/api v3.9.1-0.20191112184635-86def77f6f90+incompatible
-	github.com/openshift/hive v1.0.18
+	// github.com/openshift/hive v1.1.1
```

Comment:
I think we don’t need this. As we are only using hive/apis in repo..

Reply:
it is commented ;-)